sdk/keyvault/azure-security-keyvault-keys/src/key_client.cpp

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <azure/core/credentials.hpp>
+#include <azure/core/http/http.hpp>
+#include <azure/core/http/policy.hpp>
+
+#include "azure/keyvault/keys/key_client.hpp"
+
+#include <memory>
+#include <string>
+#include <vector>
+
+using namespace Azure::Security::KeyVault::Keys;
+using namespace Azure::Core::Http;
+
+KeyClient::KeyClient(
+    std::string const& vaultUrl,
+    std::shared_ptr<Core::TokenCredential const> credential,
+    KeyClientOptions options)
+{
+  auto apiVersion = options.GetVersionString();
+
+  // Base Pipeline
+  std::vector<std::unique_ptr<HttpPolicy>> policies;
+  policies.emplace_back(std::make_unique<TelemetryPolicy>("KeyVault", apiVersion));
```

Comment:
see if you use any literals more than once, across files, if so considermoving them to a centralised place, declare as constexprs.